dist/main.js

```diff
@@ -1614,8 +1614,8 @@ function buildPprof(i) {
       const v = i.samples[m], E = [];
       let R = v.stackId;
       for (; R !== void 0; ) {
-        const { frameId: c, parentId: F } = i.stacks[R];
-        E.push(n[c].id), R = F;
+        const { frameId: c, parentId: b } = i.stacks[R];
```

Comment:
**❔ question:** ‏Why are we committing built files? :)

Reply:
For the sake of simplicity, we allow people to install this package directly from GitHub (so we don't have to publish it on NPM, given the experimental nature of it)